src/app/portlets/content-types/fields/content-type-fields-add-row/content-type-fields-add-row.component.spec.ts

```diff
@@ -0,0 +1,124 @@
+import { ContentTypeFieldsAddRowComponent } from './content-type-fields-add-row.component';
```

Comment:
You are missing a scenario here, you need to test that the correct <li> is active when you hit left or right in the keyboard...

Like if you are in ```li[0]``` and you hit right ```li[0]``` lost the focus and ```li[1]``` get the focus.